src/main/java/com/marginallyclever/makelangelo/machines/Machines.java

```diff
@@ -5,25 +5,32 @@
 import java.util.Arrays;
 
 public enum Machines {
-    MAKELANGELO_5(new Makelangelo5()),
-    MAKELANGELO_3_3(new Makelangelo3_3()),
-    MAKELANGELO_CUSTOM(new MakelangeloCustom()),
-    CARTESIAN(new Cartesian()),
-    ZARPLOTTER(new Zarplotter());
+    MAKELANGELO_5("Makelangelo 5", new Makelangelo5()),
```

Comment:
this new method has the Machines enum String and the PlotterRender.getName() as the same value.  I hope they never change?

Reply:
`PlotterRender.getName()` is replaced by this enum String :)